1 table.sql

```diff
@@ -0,0 +1,32 @@
+use master
+go
+drop database if exists BikeDB
+go
+create database BikeDB
+go
+use BikeDB
+go
+drop table if exists BikeShop
+go
+create table dbo.BikeShop(
+BikeId int not null identity primary key,
+CustomerName varchar(50) not null constraint ck_Bike_customername_cannot_be_blank check(CustomerName <> ' '),
+CustomerAddress varchar(100) not null constraint ck_Bike_customeraddress_cannot_be_blank check(CustomerAddress <> ' '),
```

Comment:
It is better practice to break up address into separate columns (address, city, state, zip) this makes querying a lot more flexible.